lib/ws_clients/algos/index.js

```diff
@@ -80,12 +76,7 @@ module.exports = class AlgosServerClient extends WSClient {
   }
 
   async isHostOpen (exID) {
-    return new Promise((resolve) => {
-      const reqID = `${nonce()}`
-      this.isOpenCallbacks[reqID] = resolve
-
-      this.send(['is.open', this.userID, exID, reqID])
-    })
+    return await this.request(['is.open', this.userID, exID])
```

Comment:
if we aren't using try catch block, return await is redundant.. you can just change this to <br /> `return this.request(['is.open', this.userID, exID])`

Reply:
'await' makes it clear to the developer that this.request is an asynchronous function without having to look at the implementation of that function